resources/community/devhubOnMinishift.md

```diff
@@ -0,0 +1,91 @@
+ #Setup Docker in localhost
+####In case you want to run Devhub on local Minishift:
+
+ Which means you will not Have a nodejs version 10 and s2i-caddy image from Redhat in 'openshift' namespace.
+ 
+ #####You will need:
+ 
+ * Minishift
+ * Docker
+ * Access permission to console.pathfinder.gov.bc/openshift namespace.
+ 
+ 
+####Step 1: Download and config Minishift
+Ensure that you download and install [VirtualBox](https://www.virtualbox.org/wiki/Downloads) before using the embedded drivers.
+Install  before install minishift.
+
+Official instruction can be found [here]( https://docs.okd.io/latest/minishift/getting-started/installing.html).
+
+
+Use the following command to install and config the latest version of the driver with Homebrew on your computer:
+```Code
+brew install docker-machine-driver-xhyve
+sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+```
+Install minishift through Homebrew:
+```code
+brew cask install minishift
+```
+The --vm-driver VirtualBox flag will need to be given on the command line each time the minishift start command is run. For example:
+```code
+$ minishift start --vm-driver virtualbox
+```
+ #### Install Docker
+ Docker will be required to pull image from Bcgov openshift and push to your local minishift
+ 
+You can download from [here](https://docs.docker.com/docker-for-mac/install/).
+Make sure that you have the Docker client binary installed on your machine.
+
+###Pull nodejs image
```

Comment:
Instead of 'Pull NodeJS Image' 

Can we modify the instructions to be agnostic of what type of image a user wants. Instead lets describe how to pull images from the production cluster to be used locally.